apps/scorecard/private/version

```diff
@@ -0,0 +1,5 @@
+{
+  "development": "1.5.20-2017.10.04.06.14",
+  "integration": "1.5.19-2017.09.15.19.54",
+  "production": "1.5.19"
+}
```

Comment:
A general question - is this file used anymore? Those version numbers look pretty out of date. If it's no longer used, can it be removed from the apps?

Reply:
Doesn't look used! Let's leave it for now, as I'd like to remove it from all the apps at once, but thanks for flagging it.

Reply:
I agree with Molly

Reply:
Sounds good. I created #792 to track this.